src/IO/InputStreamFromROOTFileForCylindricalPET.cxx

```diff
@@ -68,30 +68,76 @@ Succeeded
 InputStreamFromROOTFileForCylindricalPET::
 get_next_record(CListRecordROOT& record)
 {
-
+    auto brentry = stream_ptr->LoadTree(static_cast<Long64_t>(current_position));
```

Comment:
why do you need to declare this out of the loop? If you do, it's fine, but then I wouldn't initialise it here (as you do it below)

Reply:
Yep, this was a mistake.